tests/e2e/README.md

```diff
@@ -14,6 +14,8 @@ E2E-suites can be manually triggered in PR stage with specific command:
 
 `/test e2e-cluster_wide-auth`
 
+> NOTE: You have to wait "prow/istio-presubmit" to succeed before you can trigger e2e tests against specific commit/SHA. It needs artifacts generated by istio-presubmit.
```

Comment:
Can /test track this dependency automatically?

If not, I'd suggest the following edit.

NOTE: Please do "/test e2e-*" only after "prow/istio-presubmit" completes, as the e2e tests need the artifacts generated by istio-presubmit.